Makefile

```diff
@@ -133,6 +136,7 @@ test-e2e-s2i-python: prepare-e2e-s2i
 test-e2e-s2i-python: E2E_S2I_LANGUAGE = python
 test-e2e-s2i-python: E2E_S2I_IMAGE_TAG = task-s2i-python:latest
 test-e2e-s2i-python: E2E_S2I_PARAMS_URL = https://github.com/Kalebu/Plagiarism-checker-Python
+test-e2e-s2i-python: E2E_S2I_IMAGE_SCRIPTS_URL = image:///usr/libexec/s2i
```

Comment:
Given my previous comment, this line should be removed on all "languages" using the default settings.